facebook/Makefile

```diff
@@ -114,6 +118,17 @@ params.json: $(TODAY)
 		output cids,individual,covidalert,archive,community \
 		start_date $(YESTERDAY)
 
+params.weekly-weights.json: $(TODAY)
+	PAT=`grep fb-survey params.json | awk 'BEGIN{FS="\""}{print $$2}' | sed 's/ /_/g;s/^/-e /'`; \
+	$(PYTHON) -m delphi_utils set \
+		debug false \
+		produce_individual_raceeth true \
+		end_date $(LAST_SATURDAY) \
+		input <(find $(QUALTRICS) -maxdepth 1 -newer $(LAST_SUNDAY) -not -newer $(shell date -d "$(LAST_SATURDAY) +2 day" +"%Y-%m-%d") -type f -name "*.csv" | sort | grep $${PAT}  | tr '\n' ',' | sed 's_$(QUALTRICS)/__g;s/,$$//' ) \
```

Comment:
Don't need to load the full range of dates for the previous week since the archive contains the last 14 days of data. If the weekly weights are too delayed (more than 7 days), however, the archive won't contain enough data. Not sure how to deal with that. Should we assume that we'll manually regenerate in that case?

Reply:
Is this solvable by increasing the depth of the archive to 17 days, or is there a concern that that will have other repercussions?

Reply:
Good idea. Yep, increasing the archive size would work _but_ would also require fancier logic elsewhere in the Makefile.

Right now, the pipeline is pretty simplistic. A run [always covers the previous week](https://github.com/cmu-delphi/covidcast-indicators/blob/ndefries/add-weekly-weights-microdata/facebook/Makefile#L128-L132) only, regardless of the dates included in the newly fetched weekly weights files. So if weekly weights are available 10 days after the end of the week, the weights will be be for 2 weeks ago but the pipeline will run for 1 week ago. The week before last will be skipped.

For fetched files to change the pipeline date range, the filenames will have to be parsed in the `WEEKLY_WEIGHTS` target and passed to `params.weekly-weights.json` through `weekly-weights-pipeline`. I'll look around to see how to do that.

Reply:
Would it make sense to hold off on a generalized solution for now, and instead have it complain loudly if a new file is for a previous week?